tools/nodejs/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "duckdb",
   "main": "./lib/duckdb.js",
-  "version": "0.2.4",
+  "version": "0.3.2-dev593.0",
```

Comment:
Please don't check this in

Reply:
Hmm, think this was updated when I ran through the CI scripts for node in lieu of build instructions, specifically `./scripts/node_version.sh`. Will revert it back.